### ISSUE TITLE
New rules suggestion: space-after-combinator and space-before-combinator

### ISSUE THREAD

**Issue by denisname** [sasstools/sass-lint#991]
This request is for the enforcement of spaces before or after combinators.
Like what rules `space-after-bang`, `space-after-colon`, `space-after-comma`, `space-before-bang`, `space-before-brace` and `space-before-colon` are doing.

Documentation example for space-after-combinator:

# Space After Combinator

Rule space-after-combinator will enforce whether or not a space should be included after a combinator (+ ~ >).

## Options

* include: true/false (defaults to true)

## Example

When `include: true`, the following are allowed. When `include: false`, the following are disallowed:

```scss
.item+ .item,
.item> .item,
.item~ .item {
  border: 0;
}
```

When `include: false`, the following are allowed. When `include: true`, the following are disallowed:

```scss
.item+.item,
.item>.item,
.item~.item {
  border: 0;
}
```

**Comment by DanPurdy** [comment]
I take it you want to be more specific that just [space-around-operator](https://github.com/sasstools/sass-lint/blob/develop/docs/rules/space-around-operator.md) ?

**Comment by denisname** [comment]
I have `space-around-operator: 2` in my `.sass-lint.yml` and got no error on rules like `.item+.item`.
Furthermore, in the operators list in the doc, `docs/rules/space-around-operator.md`, the symbols `~` is not cited.
